docs/source/remote-data-services.rst

```diff
@@ -95,49 +111,14 @@ Hadoop File System
 
 The Hadoop File System (HDFS) is a widely deployed, distributed, data-local file
 system written in Java. This file system backs many clusters running Hadoop and
-Spark.
-
-HDFS support can be provided by either hdfs3_ or pyarrow_, defaulting to the
-first library installed in that order. To explicitly set which library to use,
-set ``hdfs_driver`` using ``dask.config.set``:
-
-.. code-block:: python
-
-   # Use first available option in [hdfs3, pyarrow]
-    dd.read_csv('hdfs:///path/to/*.csv')
-
-   # Use hdfs3 for HDFS I/O
-   with dask.config.set(hdfs_driver='hdfs3'):
-       dd.read_csv('hdfs:///path/to/*.csv')
-
-   # Use pyarrow for HDFS I/O
-   with dask.config.set(hdfs_driver='pyarrow'):
-       dd.read_csv('hdfs:///path/to/*.csv')
+Spark. HDFS support can be provided by pyarrow_.
```

Comment:
```suggestion
Spark. HDFS support can be provided by PyArrow_.
```

Given that we're capitalizing Hadoop, Spark, and HDFS, we should probably capitalize PyArrow as well to show equal respect.